mindsdb/integrations/handlers/chromadb_handler/chromadb_handler.py

```diff
@@ -305,7 +305,7 @@ def select(
             TableField.ID.value: ids,
             TableField.CONTENT.value: documents,
             TableField.METADATA.value: metadatas,
-            TableField.EMBEDDINGS.value: embeddings.tolist(),
+            TableField.EMBEDDINGS.value: embeddings,
```

Comment:
this was added in my an external contributor and it breaks the rag_pipeline as embeddings is already a List